AgendaUWP/AgendaUWP/ViewModels/ContactFormPageViewModel.cs

```diff
@@ -57,7 +68,14 @@ private void InitilizeCommands()
 
         private void SaveContact()
         {
-            contactService.Insert(Contact);
+            if (IsEdit)
```

Comment:
The method "SaveContact" has two responsibilities. Could you please extract a new method? What do you think?